src/Message/style.css

```diff
@@ -0,0 +1,48 @@
+@import 'antd/lib/message/style/index.css';
```

Comment:
Missing default style import. Always check `css.js` to make sure you import all styles before starting to work on your component. Adding/reordering styles after PD approval will require another approval from PD in case the style broke